main/src/observables/conserved_gpu.cu

```diff
@@ -91,6 +91,32 @@ conservedQuantitiesGpu(Tt cv, const Tc* x, const Tc* y, const Tc* z, const Tv* v
     return {0.5 * get<0>(result), get<1>(result), get<2>(result), get<3>(result)};
 }
 
+template<class Tc, class Tv>
+struct MachSum
```

Comment:
I would call this Mach2Sum or MachSqSum, as we are in fact calculating the squared value of the Mach number.